app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivity.java

```diff
@@ -60,11 +66,24 @@ public void onBackPressed() {
     }
 
     @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.main, menu);
+        MenuItem useUtmItem = menu.findItem(R.id.action_use_utm);
```

Comment:
I would extract it to another method, like `InitUtm`. So the method `OnCreateOptionsMenu` would call different methods of menu initialization.